app/representers/concerns/embeds.rb

```diff
@@ -63,14 +63,14 @@ def embed(name, options={})
           no_curies: true,
           item_class: options.delete(:item_class),
           url: options.delete(:url),
-          item_decorator: options.delete(:item_decorator),
-          per: options.delete(:per) || Kaminari.config.default_per_page
+          item_decorator: options.delete(:item_decorator)
         }
+        getter_per = options.delete(:per) || Kaminari.config.default_per_page
+        getter_name = options.delete(:get) || name
         options[:getter] ||= ->(*) do
           # set # per page based on default, option value integer, or special :all
-          per = opts.delete(:per)
-          per = self.send(name).count if per == :all
-          PagedCollection.new(self.send(name).page(1).per(per), nil, opts.merge({parent: self}))
+          per = send(getter_name).count if getter_per == :all
+          PagedCollection.new(send(getter_name).page(1).per(per), nil, opts.merge(parent: self))
```

Comment:
see above.
seems like you are trying to have the `name` be one thing, and the `getter` another, which is fine, but I think the `:as` option already does this.